packages/open-next/src/core/routing/util.ts

```diff
@@ -356,6 +356,11 @@ export async function revalidateIfRequired(
  * @__PURE__
  */
 export function fixISRHeaders(headers: OutgoingHttpHeaders) {
+  const regex = /s-maxage=(\d+)/;
```

Comment:
nit: do not capture if there is no need (it is slower)
```suggestion
  const regex = /s-maxage=\d+/;
```